### PR TITLE
[cisco_ios] Fix grok to be aware of fman_fp_image

### PR DESCRIPTION
**Pull request by LaZyDK**
## What does this PR do?

Fix grok to allow string fman_fp_image to be optionally included.
Added tests.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #6919
- Relates #6680

### CODE REVIEW

File: packages/cisco_ios/data_stream/log/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -120,7 +120,7 @@ processors:
       field: message
       tag: grok_message
       patterns:
-        - "%{DATA:cisco.ios.facility}-%{POSINT:event.severity}-%{DATA:event.code}: %{GREEDYDATA:message}"
+        - "%{DATA:cisco.ios.facility}-%{POSINT:event.severity}-%{DATA:event.code}: (\\w+\\d+(\\/\\d+)?\\: fman_fp_image\\: )?%{GREEDYDATA:message}"
```
Comment (efd6) [inline packages/cisco_ios/data_stream/log/elasticsearch/ingest_pipeline/default.yml:123]: Do we know `\\w+\\d+(\\/\\d+)?` based on more than the examples? From the things I found above, it seems that this is too tight.

Comment (LaZyDK) [reply packages/cisco_ios/data_stream/log/elasticsearch/ingest_pipeline/default.yml]: More examples added. That part is still good.

**Comment by LaZyDK** [comment]
Added more tests from logs found online. Took into account that spaces might be more than one, and that fman_fp_image is not the only possible string.

**Review by efd6 — APPROVED** [review]
Thanks

**Pull request merged**